### PR TITLE
FD-26662: clean up feature flag

### PR DESCRIPTION
**Pull request by thombergs**

### CODE REVIEW

File: src/sync/commits.ts [hunk 1]
```diff
@@ -37,10 +36,6 @@ export default async (github: GitHubAPI, repository: Repository, cursor?: string
 		commitsData = await getCommits(true);
 	} catch (err) {
 
-		if (!await booleanFlag(BooleanFlags.RETRY_WITHOUT_CHANGED_FILES, false)) {
-			throw err;
-		}
```
Comment (thombergs) [inline src/sync/commits.ts:42]: The FF is currently enabled on all environments, which means we can just remove these three lines.

Comment (KAbakumovAtl) [reply src/sync/commits.ts:42]: Do we have any tests from where we need to remove it?

**Review by KAbakumovAtl — DISMISSED** [review]

File: src/sync/commits.ts (hunk 1 above)
Comment (thombergs) [reply src/sync/commits.ts:42]: no, those were the wild west days... :(

Comment (thombergs) [reply src/sync/commits.ts:42]: no, those were the wild west days... :(

**Review by mboudreau — APPROVED** [review]

**Review by mboudreau — APPROVED** [review]

**Review by bgvozdev — APPROVED** [review]

**Pull request merged**